### PR TITLE
Add limiter_func to interface and pass to make_operator_fluid_states

### PR DESCRIPTION
**Pull request by MTCam**
Currently, multiphysics runs with mixtures are sending unlimited fluid states to the fluid operators.   This change set fixes that issue.

**Questions for the review**:
- [x] Is the scope and purpose of the PR clear?
  - [x] The PR should have a description.
  - [x] The PR should have a guide if needed (e.g., an ordering).
- [x] Is every top-level method and class documented? Are things that should be documented actually so?
- [x] Is the interface understandable? (I.e. can someone figure out what stuff does?) Is it well-defined?
- [x] Does the implementation do what the docstring claims?
- [x] Is everything that is implemented covered by tests?
- [x] Do you see any immediate risks or performance disadvantages with the design? Example: what do interface normals attach to?

### CODE REVIEW

**Review by anderson2981 — APPROVED** [review]
Looks good to me.

**Review by majosm — APPROVED** [review]
Good catch. Thanks!

File: mirgecom/multiphysics/thermally_coupled_fluid_wall.py [hunk 1]
```diff
@@ -1165,7 +1165,7 @@ def coupled_ns_heat_operator(
     limiter_func:
 
         Callable function to be passed to
-        :func:`~mirgecom.gas_model.make_fluid_operator_states`
```
Comment (MTCam) [inline mirgecom/multiphysics/thermally_coupled_fluid_wall.py]: I like the misspelled name better.

**Comment by tulioricci** [comment]
Should we also include the limiter here?
https://github.com/illinois-ceesd/mirgecom/blob/dc19f7c98fa6003e870737d4582472eee9c4f56f/mirgecom/navierstokes.py#L436

**Pull request merged**